src/log.py

```diff
@@ -173,6 +173,20 @@ def log_show(
         share
     """
 
+    # Set up path with correct value if 'last' or 'last-X' magic keywords are used
+    last = re.match(r"last(?:-(?P<position>[0-9]{1,6}))?$", path)
```

Comment:
We probably want to match the beginning of the line (with a `^`) to avoid the case where the path ends with "last" for some reason?
```suggestion
    last = re.match(r"^last(?:-(?P<position>[0-9]{1,6}))?$", path)
```

(Not tested yet)

Reply:
`re.match` already searches if the correspondence is at the beggining of the string unlike `re.search`. The `^` isn't necessary for `re.match` but we can use `re.search` instead and add it if you think it's more suitable.

Reply:
No it's perfect like that then